firestore-translate-text/functions/src/logs/messages.ts

```diff
@@ -21,10 +21,7 @@ export const messages = {
   ],
   inputFieldNameIsOutputPath: () =>
     "The `Input` field name must not be the same as an `Output` path for this extension to function correctly",
-  start: (config = {}) => [
-    "Started execution of extension with configuration",
-    config,
-  ],
+  start: () => "Started execution of extension with configuration",
```

Comment:
Wording here should be adjusted (` with configuration`) if the config has been moved to another fn